app/views/public_account_shares/_objective.html.slim

```diff
@@ -3,10 +3,10 @@ tr
   td
     = objective.amount
     br
-    = "it takes #{([objective.amount, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).ceil} weeks"
+    = "it takes #{([objective.amount, 0].max / public_account.automatic_topup_configs.sum(:amount).+(0.1)).ceil} weeks"
```

Comment:
we need to add tests for this. specs for the controller with the rendered template would be enough. please add a ticket to do that